api/package.json

```diff
@@ -1,41 +1,41 @@
 {
-  "name": "comfygure",
-  "version": "0.1.2",
-  "license": "MIT",
-  "private": true,
-  "dependencies": {
-    "co": "~4.6.0",
-    "co-postgres-queries": "~0.2.0",
-    "config": "~1.30.0",
-    "object-hash": "~1.3.0",
-    "slug": "~0.9.1",
-    "uuid": "~3.2.1"
-  },
-  "devDependencies": {
-    "babel-eslint": "~7.1.1",
-    "babel-loader": "~6.4.1",
-    "babel-plugin-add-module-exports": "~0.2.1",
-    "babel-plugin-transform-async-to-generator": "~6.24.1",
-    "babel-plugin-transform-object-rest-spread": "~6.26.0",
-    "babel-plugin-transform-runtime": "~6.23.0",
-    "babel-polyfill": "~6.26.0",
-    "babel-preset-env": "~1.6.1",
-    "babel-register": "~6.26.0",
-    "babel-runtime": "~6.26.0",
-    "db-migrate": "~0.10.7",
-    "db-migrate-pg": "~0.3.1",
-    "eslint": "~4.19.1",
-    "eslint-config-airbnb": "~16.1.0",
-    "eslint-plugin-import": "~2.9.0",
-    "eslint-plugin-jsx-a11y": "~6.0.3",
-    "eslint-plugin-react": "~7.7.0",
-    "jest": "~22.4.3",
-    "json-loader": "~0.5.4",
-    "request": "~2.85.0",
-    "serverless": "~1.26.1",
-    "serverless-offline": "~3.18.0",
-    "serverless-webpack": "~5.1.1",
-    "webpack": "~3.11.0",
-    "webpack-config-plugin": "~1.0.0"
-  }
+    "name": "comfygure",
+    "version": "0.1.2",
+    "license": "MIT",
+    "private": true,
+    "dependencies": {
+        "co": "~4.6.0",
+        "co-postgres-queries": "~0.2.0",
+        "config": "~2.0.1",
+        "object-hash": "~1.3.0",
+        "slug": "~0.9.1",
+        "uuid": "~3.3.2"
+    },
+    "devDependencies": {
+        "@babel/core": "~7.1.2",
+        "@babel/plugin-transform-async-to-generator": "~7.1.0",
+        "@babel/preset-env": "~7.1.0",
+        "babel-core": "~7.0.0-bridge.0",
```

Comment:
Why do you keep old babel-core?

Reply:
It's for jest! It needs this version too (but the special version `bridge`)
https://jestjs.io/docs/en/getting-started#using-babel